docs/tutorial/stable-api.txt

```diff
@@ -43,10 +43,9 @@ Strict API
 ----------
 
 By default, declaring an API version guarantees behavior for commands that are
-part of the versioned API, but does not forbid using commands that are not part
+part of the stable API, but does not forbid using commands that are not part
 of the API version. To only allow commands and options that are part of the
-versioned API, specify the ``strict`` option when creating the
-specify the ``strict`` option when creating the
```

Comment:
This appears to have been a typo dating back to the original commit for this page (21e7ecde42e45d98ced06c6818495289c1e2e916).